test/run.sh

```diff
@@ -7,10 +7,21 @@ if [[ ${CI} == "true" && "${CIRCLE_JOB}" != "x86_64" ]]; then
 fi
 
 # Install necessary additional components for testing
-apt-get -qq install dnsutils -y > /dev/null
+apt-get -qq install dnsutils libcap2-bin -y > /dev/null
 
-# Create necessary directories
+# Create pihole user
+useradd -m -s /bin/bash pihole
+
+# Create necessary directories and files
 mkdir -p /etc/pihole /var/run/pihole /var/log
+touch /var/log/pihole-FTL.log
+chown pihole:pihole /etc/pihole /var/run/pihole /var/log/pihole-FTL.log
+
+# Copy binary into a location the new user pihole can access
+cp ./pihole-FTL /home/pihole
+chmod +x /home/pihole/pihole-FTL
+# Note: We cannot add CAP_NET_RAW and CAP_NET_ADMIN at this point
```

Comment:
Why is this?

Reply:
I'm not entirely sure, but I observed that, whenever I include one of these two capabilities, the `setcap` command will error with:
```
setcap: Operation not permitted.
```